Description: Drop enum34 as it is part of the standard library for python3.
             This patch prevents py3 tests from failing with:
               pkg_resources.DistributionNotFound: The 'enum34' distribution was
               not found and is required by the application
Author: Corey Bryant <corey.bryant@ubuntu.com>
Forwarded: No

--- a/requirements.txt
+++ b/requirements.txt
@@ -7,7 +7,6 @@
 oslo.utils>=3.33.0 # Apache-2.0
 oslo.concurrency>=3.25.0 # Apache-2.0
 stevedore>=1.20.0 # Apache-2.0
-enum34>=1.0.4;python_version=='2.7' or python_version=='2.6' or python_version=='3.3' # BSD
 
 eventlet!=0.18.3,!=0.20.1,<0.21.0,>=0.18.2 # MIT
 six>=1.10.0 # MIT
